matrix_client/client.py

```diff
@@ -379,3 +379,27 @@ def get_user(self, user_id):
         """
 
         return User(self.api, user_id)
+
+    def get_public_rooms_ids(self):
```

Comment:
['chunk']  should be a list of *some* of the public rooms (python dicts, not raw json).

From looking [at the spec](https://matrix.org/docs/spec/client_server/r0.2.0.html#get-matrix-client-r0-publicrooms), there is a whole load of information we're just ignoring which would be useful to almost any type of application, because then we don't have to send off a separate request to find out more information. *Ideally*, it would be nice to write a class that each room in the chunk can fill so developers get nice objects to work with. Something like ``PublicRoomListing``. I don't mind if you choose to just return the chunk however.

The other thing you need to be careful of is that it doesn't return all the rooms, you'd need to paginate to retrieve the lot so that's something we shouldn't be ignoring. Potentially a user could be testing this and getting confused as to why some of their rooms aren't coming up. Picking matrix.org as an example, it's *very* likely you're only going to get a tiny subset of the avaliable public rooms so I suggest you add a argument that allows people to paginate.



['chunk']  should be an object which contains information about a room.
Ideally the api module will return this as the raw object and the client module will return a Room object. The latter you've not implemented here so I wouldn't worry about that for now.